cfme-performance/utils/providers.py

```diff
@@ -286,6 +287,20 @@ def add_provider(provider):
 
     logger.debug('Added Provider: {}, Response: {}'.format(provider['name'], response))
 
+    # Workaround for Bug: https://bugzilla.redhat.com/show_bug.cgi?id=1351253
+    if provider['type'] == "ManageIQ::Providers::Microsoft::InfraManager":
+        print "In if statement!!!"
+        time.sleep(10)
```

Comment:
Occasionally, the provider would not be fully processed by the API and added to the vmdb, by the time it reached this step of execution